include/boost/geometry/strategies/spherical/area_huiller.hpp

```diff
@@ -117,6 +123,8 @@ public :
             calculation_type const half = 0.5;
             calculation_type const two = 2.0;
             calculation_type const four = 4.0;
+            calculation_type const pi
+                = geometry::math::pi<calculation_type>();
```

Comment:
I've done it because of the convention used. Any of the above variables is not needed. And the compiler should optimize them out anyway.